roles/pulp-webserver/templates/nginx.conf.j2

```diff
@@ -75,7 +75,7 @@ http {
             proxy_pass http://pulp-api;
         }
 
-        include conf.d/*.conf;
+        include pulp/*.conf;
```

Comment:
The Debian family has a different /etc/ filesystem layout for nginx. Looking at their layout (in a container for example) and our config files, are you confident this will work?

Reply:
I just created `pulp` dir at the same level as `conf.d`, if the filesystem is different, it was failing before:
https://github.com/pulp/pulp_installer/blob/master/roles/pulp-webserver/tasks/nginx.yml#L53
https://github.com/pulp/pulp_installer/blob/master/roles/pulp-webserver/templates/nginx.conf.j2#L78

Reply:
Debian seems to be on the same path:
https://wiki.debian.org/Nginx/DirectoryStructure

Reply:
I think you misunderstood my question.

I looked at the Fedora container's vs the debian container's folder structures. I see no reason why a /etc/nginx/pulp/ directory would not be recognized on Debian or interfere with Debian's utilities that maintain its nginx symlinks (which manages the contents of the "enabled" dirs, based on the contents of the "available" dirs), so it's safe to assume that it will function correctly. That's what I was asking.

See the file structure below for future reference:

```
root@debian-10:
/etc/nginx# ls -1p
conf.d/
fastcgi.conf
fastcgi_params
koi-utf
koi-win
mime.types
modules-available/
modules-enabled/
nginx.conf
proxy_params
scgi_params
sites-available/
sites-enabled/
snippets/
uwsgi_params
win-utf
```

```
[root@fedora-31 /etc/nginx]# ls -1p
conf.d/
default.d/
fastcgi.conf
fastcgi.conf.default
fastcgi_params
fastcgi_params.default
koi-utf
koi-win
mime.types
mime.types.default
nginx.conf
nginx.conf.default
scgi_params
scgi_params.default
uwsgi_params
uwsgi_params.default
win-utf
```

Reply:
Yeah, I really misunderstood, now it is clear to me, thanks!